docs/DeveloperGuide.md

```diff
@@ -347,13 +347,53 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
```

Comment:
Recall that I mentioned in the tutorial, that some of your high priority user stories may not be that important after all. You should focus on CRUD first, as these features make up the basic form of your product.